### PR TITLE
EASY-1790. Log naar stdout or journal-logger voor journalctl

### PR DESCRIPTION
**Pull request by janvanmansum**
Fixes EASY-1790

- [x] Process comments by @rvanheest 

#### When applied it will
* Add configuration for journalctl to the service log config.

### CODE REVIEW

File: src/main/resources/archetype-resources/src/main/assembly/dist/cfg/logback-service.xml [hunk 1]
```diff
@@ -11,8 +11,11 @@
             <pattern>[%date{ISO8601}] %-5level %msg%n</pattern>
         </encoder>
     </appender>
+    <appender name="journal" class="org.gnieh.logback.SystemdJournalAppender" />
```
Comment (rvanheest) [inline src/main/resources/archetype-resources/src/main/assembly/dist/cfg/logback-service.xml]: you want all the default configuration? https://github.com/gnieh/logback-journal#configuration contains all sorts of configuration...

File: src/main/resources/archetype-resources/src/main/assembly/dist/cfg/logback-service.xml [hunk 2]
```diff
@@ -11,8 +11,11 @@
...
     <root level="warn">
         <appender-ref ref="FILE" />
     </root>
-    <logger name="nl.knaw.dans.easy" level="info" />
+    <logger name="nl.knaw.dans.easy" level="info" >
+        <appender-ref ref="journal" />
```
Comment (rvanheest) [inline src/main/resources/archetype-resources/src/main/assembly/dist/cfg/logback-service.xml]: can't this appender go into `<root>` as well, just as the `FILE` appender?

**Comment by rvanheest** [comment]
keep in mind that this must be applied to all other projects as well. Maybe add that task to [EASY-1823](https://drivenbydata.atlassian.net/browse/EASY-1823)

File: src/main/resources/archetype-resources/src/main/assembly/dist/cfg/logback-service.xml (hunk 2 above)
Comment (janvanmansum) [reply src/main/resources/archetype-resources/src/main/assembly/dist/cfg/logback-service.xml]: 👍

**Comment by janvanmansum** [comment]
See https://drivenbydata.atlassian.net/browse/EASY-1826

**Comment by rvanheest** [comment]
awesome! can be done in one go...

**Pull request merged**